source/contracts/extensions/MarketExtensions.sol

```diff
@@ -71,22 +75,22 @@ contract MarketExtensions {
         int256 _payoutStake = int256(_reportingToken.totalSupply());
 
         IDisputeBond _designatedDisputeBond = _market.getDesignatedReporterDisputeBondToken();
-        IDisputeBond _limitedDisputeBond = _market.getLimitedReportersDisputeBondToken();
-        IDisputeBond _allDisputeBond = _market.getAllReportersDisputeBondToken();
+        IDisputeBond _firstDisputeBond = _market.getFirstReportersDisputeBondToken();
+        IDisputeBond _lastDisputeBond = _market.getLastReportersDisputeBondToken();
```

Comment:
I would still like a better name for the reporting phases.  This name makes it seem like `_firstDisputeBond` is the first dispute bond, but it isn't since the designated reporter dispute bond comes first.

Reply:
`_secondDisputeBond`?

Reply:
`initialReporting` & `finalReporting`?

Reply:
I'm 👍 on your suggestion in #dev for `round1` and `round2`.  This expands easily if we decide to add more rounds.  Alternatively, we _could_ create an array for each of these things and a constant that defines its size.  That may be over-engineering though, your call.